src/lib/components/ScriptureViewSofria.svelte

```diff
@@ -56,7 +56,26 @@ LOGGING:
     export let font: string;
     export let proskomma: SABProskomma;
 
-    $: scriptureLogs = $userSettings['scripture-logs'] ? {"root": 1, "docResult": 1, "document":1, "paragraph": 1, "phrase" :1 , "chapter": 1, "verses": 1, "text": 1, "sequence": 1, "wrapper":1, "milestone":1, "blockGraft": 1, "inlineGraft": 1, "mark": 1, "meta": 1, "row": 1} : $logs['scripture'];
+    $: scriptureLogs = $userSettings['scripture-logs']
```

Comment:
You PR should not have changes to ScriptureViewSofria.svelte in it.  This looks to be from another PR?

Reply:
I accidently committed all the changes for Issue #516 and #566 into this one pull request.